Makefile

```
@@ -71,13 +71,13 @@ ifdef spike-tandem
     endif
 endif
 
+# target takes one of the following cva6 hardware configuration:
+# cv64a6_imacfd_sv39, cv32a6_imac_sv0, cv32a6_imac_sv32, cv32a6_imacf_sv32
+target     ?= cv64a6_imacfd_sv39
```

Nitpick: The order should me imafd[c] because `imafd` is `g` and then `c` is optional to that. I remember something when reading the official RISC-V spec.

```suggestion
target     ?= cv64a6_imafdc_sv39
```

---

Sorry to be so annoying, I know that means changing all the file names as well :(

---

Dont worry, I have modified the names